bindata/v3.11.0/kube-apiserver/pod.yaml

```diff
@@ -13,10 +13,21 @@ spec:
     - name: fix-audit-permissions
       image: ${IMAGE}
       imagePullPolicy: IfNotPresent
-      command: ['sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
+      command: ['/bin/sh', '-c', 'chmod 0700 /var/log/kube-apiserver']
       volumeMounts:
         - mountPath: /var/log/kube-apiserver
           name: audit-dir
+    - name: wait-for-port
+      image: ${IMAGE}
+      imagePullPolicy: IfNotPresent
+      command: ['/usr/bin/timeout', '90', "/bin/bash", "-c"]
+      args:
+      - |
+        echo -n "Waiting for port :6443 to be released."
```

Comment:
remember that BZ that was asking us to make this port configurable?

Reply:
When we implement that, we make this more generic here too. Probably the whole thing will turn into a template with port variable.